### PR TITLE
Add sync options

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

File: .github/workflows/unit_tests.yaml [hunk 1]
```diff
@@ -27,11 +27,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        foreman_version: [2_5, 3_1]
+        foreman_version: [6.0.43]
```
Comment (m-bucher) [inline .github/workflows/unit_tests.yaml:30]: That does not look like a `foreman_version` any more :smile:

File: app/models/scc_account.rb [hunk 2]
```diff
@@ -13,6 +13,11 @@ class SccAccount < ApplicationRecord
   MONTHLY = 'monthly'.freeze
   TYPES = [NEVER, DAILY, WEEKLY, MONTHLY].freeze
 
+  # MIRRORING_POLICY_COMPLETE doesn't work for suse repository because deltarpm metadata is not supported by pulp3
+  SCC_MIRRORING_POLICY = ::Katello::RootRepository::MIRRORING_POLICIES.dup
+  SCC_MIRRORING_POLICY.delete(::Katello::RootRepository::MIRRORING_POLICY_COMPLETE)
+  SCC_MIRRORING_POLICY.freeze
```
Comment (m-bucher) [inline app/models/scc_account.rb:19]: Nitpicky, but should it not be `SCC_MIRRORING_POLICIES`!?

**Review by m-bucher — CHANGES_REQUESTED** [review]
Apart from the small suggestions, this looks good.

**Review by m-bucher — APPROVED** [review]

File: app/models/scc_account.rb [hunk 3]
```diff
@@ -278,4 +285,18 @@ def invalidate_subscription_status(items_to_invalidate, save_record:)
     end
     items_to_invalidate
   end
+
+  def self.download_policy_selection_values
+    names = { 'on_demand' => _('On Demand'), 'immediate' => _('Immediate') }
+    ::Katello::RootRepository::DOWNLOAD_POLICIES.map do |p|
+      names.include?(p) ? [names[p], p] : [p, p]
```
Comment (ekohl) [inline app/models/scc_account.rb:292]: I was reading through this and a more efficient way is to use [fetch](https://ruby-doc.org/core-3.0.1/Array.html#method-i-fetch) with a default value:
```suggestion
      [names.fetch(p, p), p]
```

File: app/models/scc_account.rb [hunk 4]
```diff
@@ -278,4 +285,18 @@ def invalidate_subscription_status(items_to_invalidate, save_record:)
     end
     items_to_invalidate
   end
+
+  def self.download_policy_selection_values
+    names = { 'on_demand' => _('On Demand'), 'immediate' => _('Immediate') }
```
Comment (ekohl) [inline app/models/scc_account.rb:290]: Looking at https://github.com/Katello/katello/blob/28a7171ed3be4fdb0ebd96bfad2304b05f0ea080/app/helpers/katello/concerns/smart_proxy_helper_extensions.rb#L28-L49 you should use constants such as `::Katello::RootRepository::DOWNLOAD_ON_DEMAND` and `::Katello::RootRepository::DOWNLOAD_IMMEDIATE`.

**Review by ekohl — COMMENTED** [review]
Bit late to the party, but I was reading through the diff because of a packaging update.

**Pull request merged**